### PR TITLE
Documentation: fix installation guide

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
- update instructions about how to download Github actions artifacts

### CODE REVIEW

File: Documentation/install.md [hunk 1]
```diff
@@ -4,25 +4,18 @@
...
-```
-$ inspektor-gadget version
-v0.1.0-alpha.2-12-gb9fd574
-```
+You can find other releases on [releases](https://github.com/kinvolk/inspektor-gadget/releases).
 
-### From a release
-
-Follow instructions from the [releases](https://github.com/kinvolk/inspektor-gadget/releases).
+### Latest version
```
Comment (alban) [inline Documentation/install.md:18]: this section seems empty (or the next line should have 4 `#`).

File: Documentation/install.md [hunk 2]
```diff
@@ -31,25 +24,35 @@ Follow instructions from the [releases](https://github.com/kinvolk/inspektor-gad
...
-- if you wish to make changes to traceloop program, update `gadget-ds/gadget.Dockerfile` to pick your own image of traceloop.
-- if you wish to make other changes in the gadget container image, update `Makefile` to choose the default `gadgetimage`.
-
-See the [minikube](#Development-environment-on-minikube-for-the-traceloop-gadget) section for a faster development cycle.
+- the compilation uses `tools/image-tag` to choose the tag of the container
+image to use according to the branch that you are compiling.
+- you can push the docker images to another registry and use the `--image`
+argument to choose them as described below.
+- if you wish to make changes to traceloop program, update
+`gadget-ds/gadget.Dockerfile` to pick your own image of traceloop.
```
Comment (alban) [inline Documentation/install.md:50]: I guess you will have to update gadget-ds -> gadget-container if your other PR gets in first.

File: Documentation/install.md (hunk 1 above)
Comment (mauriciovasquezbernal) [reply Documentation/install.md:18]: Right, there was a missing `#` on the next line.

File: Documentation/install.md (hunk 2 above)
Comment (mauriciovasquezbernal) [reply Documentation/install.md:50]: I propose this to go first, as the other have some conflicts and require a little bit more time.

Comment (alban) [reply Documentation/install.md:50]: SGTM

**Pull request merged**